core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -243,13 +243,13 @@ public long commit() {
     }
 
     @Override
-    public long commitWithLag() {
-        return commit();
+    public void ic() {
```

Comment:
Not descriptive enough method name

Reply:
it is memorable though, don't you agree? :)

And documented:

![image](https://user-images.githubusercontent.com/7276403/201938537-669ec485-6024-41e2-83c4-e131263c2055.png)

Reply:
Then it's inconsistent with `commit` method name, it should be `c` then.

Reply:
i'm happy to rename :)

Reply:
seriously however, `intremediateCommit()` is hard to spell, 9 times out of 10 I miss-spell _intermediate_